docs/cli-spec.md

```diff
@@ -0,0 +1,47 @@
+# Signing CLI specficiation
+
+### Inputs
+
+The parameters to the signing client are as follows:
+
+`sign code AzureKeyVault [options] <files>`
+
```

Comment:
Could we have [Synopsis](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-sign#synopsis) section like `dotnet nuget sign`?